checks.d/process.py

```diff
@@ -241,8 +245,33 @@ def get_process_state(self, name, pids):
             st['r_bytes'].append(ioinfo.get('read_bytes'))
             st['w_bytes'].append(ioinfo.get('write_bytes'))
 
+            pagefault_stats = self.get_pagefault_stats(pid)
+            if pagefault_stats is not None:
+                (minflt, cminflt, majflt, cmajflt) = pagefault_stats
+                st['minflt'].append(minflt)
+                st['cminflt'].append(cminflt)
+                st['majflt'].append(majflt)
+                st['cmajflt'].append(cmajflt)
+
         return st
 
+    def get_pagefault_stats(self, pid):
+        if not Platform.is_linux():
+            return None
+
+        def file_to_string(path):
+            with open(path, 'r') as f:
+                res = f.read()
+            return res
+
+        # http://man7.org/linux/man-pages/man5/proc.5.html
+        try:
+            data = file_to_string('/proc/%s/stat' % pid)
+        except:
+            return None
```

Comment:
we should log something when this happens. A `debug` level is probably enough though